addons/opacity-slider/addon.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Opacity slider",
+  "description": "Adds an opacity slider to the costume editor color picker.",
+  "tags": ["editor", "costumeEditor", "beta", "featured"],
+  "info": [
+    {
+      "type": "info",
+      "text": "This doesn't work on bitmap mode.",
```

Comment:
You should update this to specify the other locations where the opacity slider is disabled.